docs/addons/gmaestro.md

```diff
@@ -0,0 +1,100 @@
+1. **Install gMaestro** - As a single pod on your cluster with a single command line.
+2. **Multi-cluster visibility** - Gain visibility into each controller, resource request, container, HPA policy, utilization, and cost.
+3. **Apply custom recommendations** - gMaestro generates custom recommendations by analyzing the actual utilization of each controller.
+
+For additional information, visit [gMaestro documentation](https://gmaestro.gitbook.io/gmaestro-docs/).
+
+## Prerequisite 
+Before using gMaestro, you need to:
+1. [Sign up](https://app.granulate.io/gMaestroSignup) to the gMaestro platform
+2. Download a sample YAML file - After signing up to gMaestro, navigate to the [Deploy](https://app.granulate.io/deploy) on the left-hand menu, fill in the required fields and click on "Generate Config File" 
+
+
+## Installation
+
+Using [npm](https://npmjs.org):
+
+```bash
+$ npm i @granulate/gmaestro-eks-blueprints-addon
+```
+
+## Usage
+
+```typescript
+import 'source-map-support/register';
+import * as CDK from 'aws-cdk-lib';
+import * as blueprints from '@aws-quickstart/eks-blueprints';
+import {GmaestroAddOn} from '@granulate/gmaestro-eks-blueprints-addon';
+
+const app = new cdk.App();
+
+const addOn = new GmaestroAddOn({
+        clientIdSecretName: "<client id secret name>", // Create and copy from gMaestro deployment yaml
+        clientName: "<client name>",
+        clusterName: "<cluster name>",
+        namespace: "<namespace>",
+        grafanaMetricsSecretName: "<grafana metrics secret name>", // Create and copy from gMaestro deployment yaml
+        grafanaLogsSecretName: "<grafana logs secret name>", // Create and copy from gMaestro deployment yaml
+    });
+
+const blueprint = blueprints.EksBlueprint.builder()
+  .addOns(addOn)
+  .build(app, 'my-stack-name');
+```
+
+Use the following command to validate that gMaestro installed successfully:
+
+```bash
+$ kubectl get pods --all-namespaces | grep granulate-maestro
+
+NAMESPACE     NAME                                 READY   STATUS    RESTARTS   AGE
+default       granulate-maestro-6947dc87bc-k5nfc   2/2     Running   0          11m
+kube-system   aws-node-9rhgx                       1/1     Running   0          16m
+kube-system   coredns-d5b9bfc4-8v8k5               1/1     Running   0          21m
+kube-system   coredns-d5b9bfc4-r5sdb               1/1     Running   0          21m
+kube-system   kube-proxy-js5pn                     1/1     Running   0          16m
+```
+
+After a few seconds, you will gain full visibility into your K8s cluster objects.
+The first rightsizing recommendations may take up to 5 minutes to load.
+
+## `gMaestroAddOn` Required (props)
+Take the following parameter from the sample YAML file that was downloaded.
+
+#### `clientIdSecretName: string`
+
+Create a secret (as a plaintext) in AWS copy its value from the Deployment section `MAESTRO_CLIENT_ID` 
```

Comment:
Please specify in AWS Secrets Manager. There are other ways to inject the secret, i.e. through the parameter store.

Reply:
done